lexer_test.go

```diff
@@ -132,7 +145,6 @@ var lexerWrongTestCase = []struct {
 	pos     Pos
 	message string
 }{
-	{"?", 0, "illegal input character: '?'"},
```

Comment:
Is it hard to rewrite this test instead of removing?

Reply:
Hmm, I think there is no ASCII printable character remaining.
If we want to leave illegal character test case, we need to use unprintable character or some unicode chatecters.

I would try to use `\b`(beep character)

Reply:
https://github.com/cloudspannerecosystem/memefish/pull/182/commits/5071b3727134fc09a38f29cb72bdde561d2a5b00